lib/bundles/inspec-compliance/api.rb

```diff
@@ -36,6 +36,7 @@ def self.profiles(config)
           end
         end.flatten
       else
+        puts '401 Unauthorized. Please check your token.'
```

Comment:
Are we sure the only case where `data == ''` is a 401?  My first impression when reading this code is that I'm a bit surprised that this isn't a case statement on the http response code. Also, if possible it feels like this is a case where we would want to exit with a non-zero exit status.